<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Магнетрон</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="phywidgets::ModelViewerGLWidget" name="coubGL"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="phywidgets::PhyCoubControllerWidget" name="phyCoubController" native="true"/>
       </item>
       <item>
        <widget class="QGroupBox" name="scalarValues">
         <property name="title">
          <string>Опции</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="phywidgets::PhyScalarDoubleControllerWidget" name="radiusScalar" native="true"/>
          </item>
          <item>
           <widget class="phywidgets::PhyScalarDoubleControllerWidget" name="heightScalar" native="true"/>
          </item>
          <item>
           <widget class="phywidgets::PhyScalarDoubleControllerWidget" name="electricScalar" native="true"/>
          </item>
          <item>
           <widget class="phywidgets::PhyScalarDoubleControllerWidget" name="magneticScalar" native="true"/>
          </item>
          <item>
           <widget class="phywidgets::PhyScalarDoubleControllerWidget" name="energyScalar" native="true"/>
          </item>
          <item>
           <widget class="phywidgets::PhyScalarULongControllerWidget" name="lifeParticleCountScalar" native="true"/>
          </item>
          <item>
           <widget class="phywidgets::PhyScalarBoolControllerWidget" name="trajectory" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="restart">
   <property name="text">
    <string>Перезапустить</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>phywidgets::ModelViewerGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>ModelViewerGLWidget.h</header>
  </customwidget>
  <customwidget>
   <class>phywidgets::PhyCoubControllerWidget</class>
   <extends>QWidget</extends>
   <header>PhyCoubControllerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>phywidgets::PhyScalarDoubleControllerWidget</class>
   <extends>QWidget</extends>
   <header>PhyScalarDoubleControllerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>phywidgets::PhyScalarULongControllerWidget</class>
   <extends>QWidget</extends>
   <header>PhyScalarULongControllerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>phywidgets::PhyScalarBoolControllerWidget</class>
   <extends>QWidget</extends>
   <header>PhyScalarBoolControllerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
